Rakefile

```diff
@@ -4,15 +4,15 @@ Bundler::GemHelper.install_tasks
 require 'rake/testtask'
 
 Rake::TestTask.new do |t|
-  t.libs << "spec" << "lib"
-  t.pattern = "spec/**/*_spec.rb"
+  t.libs << 'spec' << 'lib'
```

Comment:
I'm on this side of the debate 

http://anti-pattern.com/always-use-double-quoted-strings-in-ruby